beacon_node/execution_layer/src/lib.rs

```diff
@@ -928,259 +929,99 @@ impl<T: EthSpec> ExecutionLayer<T> {
         }
     }
 
-    async fn determine_and_fetch_payload(
+    /// Fetches local and builder paylaods concurrently, Logs and returns results.
+    async fn fetch_local_and_builder_payloads(
```

Comment:
Maybe the naming of this func should be reversed, as it returns `(builder, local)` not `(local, builder)`?

```suggestion
    async fn fetch_builder_and_local_payloads(
```